src/newsletters.ts

```diff
@@ -0,0 +1,49 @@
+import type { Newsletter, Newsletters } from './types';
+
+const guardianTodayNewsletter: Newsletter = {
+	listName: 'today-uk',
+	title: 'The Guardian Today',
+	description:
+		'The day’s headlines and highlights emailed direct to you every morning. Get the whole picture from a source you trust',
+	frequency: 'Every day',
+};
+
+const guardianTodayUSNewsletter: Newsletter = {
+	listName: 'today-us',
+	title: 'The Guardian Today: US edition',
+	description:
+		'For our readers in the US, we offer a regional edition of our daily email, delivering the most important headlines and highlights every morning',
+	frequency: 'Every day',
+};
+
+const guardianTodayAUNewsletter: Newsletter = {
+	listName: 'today-au',
+	title: 'The Guardian Today: Australia edition',
+	description:
+		'For our readers in Australia, we offer a regional edition of our daily email, delivering the biggest headlines and highlights every lunchtime',
+	frequency: 'Every day',
+};
+
+const theWeekendPapers: Newsletter = {
+	listName: 'the-weekend-papers',
+	title: 'The Weekend Papers',
+	description:
+		'Be the first to find out what’s coming up in the Guardian and the Observer every weekend',
+	frequency: 'Weekly',
+};
+
+export const newsletters: Newsletters = {
```

Comment:
Am I correct that this is test data for now, awaiting further implementation?

Reply:
Correct. One of the key things I want to get working next would be setting this up with the Identity API in some format to pull this data through. If we can get this to be a truly server-side rendered project, then we can reduce the number of calls to the API right down so that it's cached in some format, though I haven't thought it through in too much detail yet.

The same will go for User info as well

Reply:
Sounds good. The newsletter data could be injected on the server, but the user data would have to be fetched asynchronously on the client as it shouldn't be cached